pylxd/models/network.py

```diff
@@ -17,18 +17,27 @@
 class Network(model.Model):
```

Comment:
A question here - I was also thinking about adding `__str__()` / `__repr__()` methods in order to display network details in human readable form (e.g. using `marshall()`). Is there any reason why it wasn't implemented?

Reply:
Yeah, I'd love for `__str__()` and `__repr__()` to be implemented, but I wasn't going to force it as an issue!  If you have the time to do it, then that would make this class more of an exemplar for the others to follow on with.